kafka_consumer/assets/configuration/spec.yaml

```
@@ -215,13 +215,22 @@ files:
           only generate a single check process, but if one host goes down,
           KafkaClient / KazooClient tries contacting the next host.
           Details: https://github.com/DataDog/dd-agent/issues/2943
+
+          You may specify a single server like:
+
+            zk_connect_str: localhost:2181
+
+          or multiple servers like:
+
+            zk_connect_str:
+            - server1:2181
+            - server2:2181
         value:
-          type: array
-          items:
-            type: object
-          example:
-            - localhost:2181
-            - <ZOOKEEPER_ENDPOINT>:2181
+          anyOf:
+          - type: array
+            items:
+              type: string
+          - type: string
```

Mind if we flip it to match `kafka_connect_str`?

```suggestion
          - type: string
          - type: array
            items:
              type: string
```